java/src/jmri/util/ImmediatePipeOutputStream.java

```diff
@@ -0,0 +1,34 @@
+package jmri.util;
+
+import java.io.IOException;
+import java.io.PipedInputStream;
+import java.io.PipedOutputStream;
+
+/**
+ * Makes a workaround for standard {@link PipedOutputStream} wait.
+ * <p>The {@link PipedInputStream#read()}, in case the receive buffer is
+ * empty at the time of the call, waits for up to 1000ms.
+ * {@link PipedOutputStream#write(int)} does call <code>sink.receive</code>,
+ * but does not <code>notify()</code> the sink object so that read's
+ * wait() terminates.
+ * </p><p>
```

Comment:
The terminating `</p>` is not needed in Javadocs, and just makes it harder to read. Can you remove these?

Reply:
I will fix that, but that actually was copied verbatim from what @svatoun wrote in #8213.